app/serializers/education_benefits_claim_serializer.rb

```diff
@@ -1,4 +1,11 @@
 # frozen_string_literal: true
 class EducationBenefitsClaimSerializer < ActiveModel::Serializer
   attributes :id, :form, :submitted_at, :regional_office, :confirmation_number
+  attribute :spool, if: proc { Rails.env.development? }
+
+  def spool
+    EducationForm::Forms::Base.build(object).text
+  rescue
```

Comment:
if there's a problem with rendering a spool file shouldn't we let it go to sentry?